### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by wandergithub**
Build website home, calculator, and quote pages. Added style.
1. [Set React Router.](https://github.com/wandergithub/Math-Magicians/commit/ac77f50b327f3a8d2392f7be090db583855857ac)
2. [Create header and set Link elements to navigate throughout each page.](https://github.com/wandergithub/Math-Magicians/commit/892a68b906d45d0b2f3ba7628a6a7298dfb94950)
3. [Add all pages contents.](https://github.com/wandergithub/Math-Magicians/commit/5954e5375d51eb0c250e201a64545eb11d345c57)
4. [Add style to website.](https://github.com/wandergithub/Math-Magicians/commit/5917c2c3c4bf36f46405c1519972c22fafc2735e)
5. [Solve linters.](https://github.com/wandergithub/Math-Magicians/commit/65be0b2fd4b3424079752df0df0e0cbd3d2fe1cb)

### CODE REVIEW

File: src/components/CalcPage.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Calculator from './Calculator';
+import Header from './Header';
+import './styles/Calculator.css';
+
+const CalcPage = () => (
+  <div>
+    <Header />
+    <div className="calculator-container">
+      <h2>Let&apos;s do some math!</h2>
+      <Calculator />
```
Comment (medaminedev66) [inline src/components/CalcPage.js:10]: - [ ] The calculator doesn't work. I recommend fixing that so that your project is complete.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @wandergithub  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- The repo looks professional ✔️
- There are no linter errors ✔️
- You have a descriptive PR ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/CalcPage.js (hunk 1 above)
Comment (wandergithub) [reply src/components/CalcPage.js:10]: Hi @medaminedev66, In which way the calculator doesn't work? 
Could you please provide extra feedback?

[Loom video doing basic operations with calculator](https://www.loom.com/share/219fbf5ea515418b85b178b19b19dd5c)

**Review by roshan-bajgain — APPROVED** [review]
Hi @wandergithub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️
- Project meets all the technical requirements.

## Optional suggestions

- [x] [Optional] whenever users click some maths numbers, it is not shown on the screen only the result is showing So I would suggest you whenever users click numbers try to display them on the screen.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wandergithub** [comment]
Thanks for the suggestion @roshan-bajgain. Just implemented It and the calculator looks better now. 👍🏽 
image

**Pull request merged**